packages/components/src/internal/components/files/FilePreviewGrid.tsx

```diff
@@ -2,17 +2,7 @@ import React from 'react';
 import { List, Map } from 'immutable';
 
 import { Alert, Grid, GridColumn, InferDomainResponse } from '../../..';
-
-export interface FileGridPreviewProps {
-    previewCount: number;
-    header?: string;
-    infoMsg?: any;
-    onPreviewLoad?: (response: InferDomainResponse, fileData?: File) => any;
-    acceptedFormats?: string; // comma-separated list of allowed extensions i.e. '.png, .jpg, .jpeg'
-    initialData?: InferDomainResponse;
-    skipPreviewGrid?: boolean;
-    errorStyle?: string;
-}
+import {FileGridPreviewProps} from "../../../public/files/models";
```

Comment:
nit: Please be sure to run `yarn run lint-branch-fix` before merging.

Reply:
sure, will do! thank you!